frontend/src/features/public-form/components/FormPaymentPage/stripe/components/DownloadReceiptBlock.tsx

```diff
@@ -82,36 +123,64 @@ export const DownloadReceiptBlock = ({
-      <Stack spacing="2rem">
+    <>
+      <Box bgColor="white" p="2rem">
         <Stack tabIndex={-1} spacing="0.75rem">
-          <Text textStyle="h2" color="content.strong">
-            Your payment has been made successfully.
+          <Text textStyle="h2" color="secondary.500">
+            Thank you, your payment has been made successfully.
           </Text>
-          <Text textStyle="subhead-1" color="content.strong">
-            Your form has been submitted and payment has been made.
+          <Text textStyle="subhead-1" color="secondary.500">
+            This is a message that the admins can customise on their thank you
+            page. This helps to make it more personable to admins. This is a
+            message that the admins can customise on their thank you page. This
+            helps to make it more personable to admins.This is a message that
+            the admins can customise on their thank you page. This helps to make
+            it more personable to admin.
           </Text>
         </Stack>
-        <Divider />
+      </Box>
+      <Box mt="2rem" px="1rem" py="2rem" bgColor="white">
         <Stack>
-          <Text textStyle="h2" mb="0.5rem" color="content.strong">
-            Payment summary
-          </Text>
-          <PaymentSummaryRow title="Product/service" input={productName} />
-          <PaymentSummaryRow title="Amount" input={formattedAmount} />
-          <PaymentSummaryRow title="Date" input={paymentTimestamp} />
-          <PaymentSummaryRow title="Response ID" input={submissionId} />
+          <Box mb="1.5rem" px="1.5rem">
+            <Text textStyle="h2" mb="0.5rem" color="secondary.500">
+              Payment details
+            </Text>
+            <PaymentDetailsRow title="Payment date" input={paymentTimestamp} />
+            <PaymentDetailsRow title="Response ID" input={submissionId} />
+          </Box>
+          <Box
+            bgColor="primary.100"
+            flexDir="row"
+            justifyContent="space-between"
+            mb="1rem"
+            py="1rem" // h="4rem"
+            px="1.5rem"
+          >
+            <Text textStyle="h2" mb="0.5rem" color="secondary.500">
+              Summary
+            </Text>
+            <Stack spacing="0.75rem" my="1rem">
+              {products.map((product) => (
+                <LineItem productItem={product} />
+              ))}
+            </Stack>
+            <Divider />
+            <Stack my="1rem">
+              <PaymentSummaryRow title="Total" input={totalAmount} />
+            </Stack>
+          </Box>
         </Stack>
-      </Stack>
 
-      <Button
-        mt="2.75rem"
-        width={{ base: '100%', md: 'auto' }}
-        leftIcon={<BiDownload fontSize="1.5rem" />}
-        onClick={handleInvoiceClick}
-      >
-        Save proof of payment
-      </Button>
-    </Box>
+        <Stack mx={{ base: '0', md: '1.5rem' }} mt="1.5rem">
+          <Button
+            width={{ base: '100%', md: '100%' }}
```

Comment:
Simliiar to above, `base` and `md` are the same, would it work if we use `w="100%"`?

Reply:
yes it works! 🙏🏼